### DISCUSSION TITLE
希望能维护下vue2版本的组件库

### DISCUSSION THREAD

**Discussion by iPandaQAQ** [zouyaoji/vue-cesium#493] [Q&A]
由于大部分项目使用的是vue2，组件库看起来没有vue3版本的好，还是希望能维护下

**Comment by zouyaoji** [comment]
你好，vue3版本的基于vue3的特性重构过了，改动很大，没精力再去兼容vue2的版本了。建议老项目维持不变，新项目用vue3了，vue官方也推荐使用vue3版本的。
